src/components/Tasks/TaskStatus.js

```diff
@@ -1,7 +1,21 @@
 import React from 'react'
 import { textStyle, GU } from '@aragon/ui'
+import dayjs from 'dayjs'
+import { datesDiff } from '../../utils/date-utils'
 
-function TaskStatus() {
+function getEndDay(dueDate) {
+  const diff = datesDiff(dayjs(), dayjs(dueDate))
+
+  if (diff === 0) {
+    return ' Due Today'
+  }
+  if (diff === 1) {
+    return 'Due In the next day'
+  }
+  return `Due In ${diff} days`
+}
+
+const TaskStatus = React.memo(function TaskStatus({ dueDate }) {
   return (
```

Comment:
Not sure if we should use the `useNow` here since we are not showing a timer just a label, and i don't think the user will be looking this screen for hours. it's more like the user will enter to the task list take a look and go to complete some takss cc @bpierre

Reply:
pinging @bpierre

Reply:
Yes I think we could set a timeout for the next refresh instead, that would re-render in `startOfNextDay - now` every time?